sdk/storage/azure-storage-blob-changefeed/src/main/java/com/azure/storage/blob/changefeed/BlobChangefeedClientBuilder.java

```diff
@@ -4,10 +4,72 @@
 package com.azure.storage.blob.changefeed;
 
 import com.azure.core.annotation.ServiceClientBuilder;
+import com.azure.core.http.HttpPipeline;
+import com.azure.storage.blob.BlobServiceAsyncClient;
+import com.azure.storage.blob.BlobServiceClient;
+import com.azure.storage.blob.BlobServiceVersion;
 
 /**
- * TEMP
+ * This class provides a fluent builder API to help aid the configuration and instantiation of
+ * {@link BlobChangefeedClient BlobChangefeedClients} and {@link BlobChangefeedAsyncClient BlobChangefeedAsyncClients}
+ * when {@link #buildClient() buildClient} and {@link #buildAsyncClient() buildAsyncClient} as called respectively.
  */
 @ServiceClientBuilder(serviceClients = {BlobChangefeedClient.class, BlobChangefeedAsyncClient.class})
-public class BlobChangefeedClientBuilder {
+public final class BlobChangefeedClientBuilder {
+    private final String accountUrl;
+    private final HttpPipeline pipeline;
+    private final BlobServiceVersion version;
+
+    /**
+     * Constructs the {@link BlobChangefeedClientBuilder} using the {@link BlobServiceClient#getAccountUrl() account
+     * URL} and {@link BlobServiceClient#getHttpPipeline() HttpPipeline} properties of the passed {@link
+     * BlobServiceClient}.
+     *
+     * @param client {@link BlobServiceClient} whose properties are used to configure the builder.
+     */
+    public BlobChangefeedClientBuilder(BlobServiceClient client) {
+        this.accountUrl = client.getAccountUrl();
+        this.pipeline = client.getHttpPipeline();
+        this.version = client.getServiceVersion();
+    }
+
+    /**
+     * Constructs the {@link BlobChangefeedClientBuilder} using the {@link BlobServiceAsyncClient#getAccountUrl()
+     * account URL} and {@link BlobServiceAsyncClient#getHttpPipeline() HttpPipeline} properties of the passed {@link
+     * BlobServiceAsyncClient}.
+     *
+     * @param client {@link BlobServiceClient} whose properties are used to configure the builder.
+     */
+    public BlobChangefeedClientBuilder(BlobServiceAsyncClient client) {
+        this.accountUrl = client.getAccountUrl();
+        this.pipeline = client.getHttpPipeline();
+        this.version = client.getServiceVersion();
+    }
+
+    /**
+     * Creates a {@link BlobChangefeedClient} based on options set in the builder.
```

Comment:
Are we planning on adding options? Or should we change the doc?

Reply:
Let me change the doc for now and add it if we do end up adding options